presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraPageSink.java

```diff
@@ -76,6 +84,10 @@ public CassandraPageSink(
         this.columnTypes = ImmutableList.copyOf(requireNonNull(columnTypes, "columnTypes is null"));
         this.generateUUID = generateUUID;
 
+        toCassandraDate = protocolVersion.toInt() <= ProtocolVersion.V3.toInt() ?
```

Comment:
do we have coverage for both protocols?

Reply:
As `cassandra.protocol-version` in [cassandra.rst](https://github.com/prestosql/presto/blob/master/presto-docs/src/main/sphinx/connector/cassandra.rst), it says we can use V2, V3 and V4. Current upstream's test coverage is only V3 though.